app/backplane/power_module/src/c_sensing_tenant.cpp

```diff
@@ -49,6 +54,13 @@ void CSensingTenant::Run() {
         i++;
     }
 
+    uint32_t tmpTimestamp = 0;
+    if (int ret = rtc.GetUnixTime(tmpTimestamp); ret < 0) {
+        LOG_ERR("Failed to get time from RTC");
+    } else {
+        timestampedData.timestamp = tmpTimestamp;
```

Comment:
consider avoiding tmpTimestamp and just pointing directly to timestampedData.timestamp - getUnixTime leaves its pointed value unchanged on error

Reply:
Added comment stating we can't do direct because of inability to use references on packed struct fields